lambdas/send_from_queue/send_from_queue.py

```diff
@@ -0,0 +1,226 @@
+import base64
+import json
+import logging
+import os
+
+import boto3
+from botocore.vendored.requests import post
+
+logger = logging.getLogger()
+
+# First log the function load message, then change
+# the level to be configured via environment variable.
+logger.setLevel(logging.INFO)
+logger.info('Loading function')
+
+# Log level is set as a string, default to 'INFO'
+log_level = os.environ.get('LOG_LEVEL', 'INFO').upper()
+numeric_level = getattr(logging, log_level, None)
+if not isinstance(numeric_level, int):
+    raise ValueError('Invalid log level: {}'.format(log_level))
+logger.setLevel(numeric_level)
+
+
+def _get_target_queue():
+    """
+    Get the target SQS name for the processed hooks from the
+    OS environment variable.
+
+    Return the name of the queue
+    """
+    queue_name = os.environ.get('TARGET_QUEUE')
+    if not queue_name:
+        raise StandardError(
+            "Environment variable TARGET_QUEUE was not set"
+        )
+
+    return queue_name
+
+
+def _get_queue_object(queue_name):
+    """
+    Connect with SQS via boto and return an object representing
+    the queue.
+    """
+    try:
+        sqs_resource = boto3.resource('sqs')
+        queue_object = sqs_resource.get_queue_by_name(QueueName=queue_name)
+    except:
+        raise StandardError(
+            "Unable to connect to the SQS queue"
+        )
+
+    return queue_object
+
+
+def _get_api_url():
+    """
+    Find the url of the API Gateway by getting its id from boto
+
+    Return the url
+    """
+    try:
+        api_client = boto3.client('apigateway')
+        api_list = api_client.get_rest_apis()
+    except:
+        logger.error(
+            "Unable to connect to the apigateway"
+        )
+
+    for api in api_list.get("items"):
+        if api.get("name") == "edx-tools-webhooks-processing":
+            api_id = api.get("id")
+            break
```

Comment:
nit/question: just because I have seen this a couple of times here, I am curious. Are we ever expecting to get a null api id? Checking the boolean-ness of it might just be excessive. Not very important, just want to know